myriapython.markdown

```diff
@@ -30,17 +30,16 @@ Users can install the Python libraries using `pip install myria-python`. Develop
 
 ### Part 1: Uploading Data
 
-We illustrate the basic functionality using examples in the directory
-`jsonQueries/getting_started`. The  `jsonQueries` directory contains additional examples. In the example below, we upload the smallTable to the Myria Service. Here is an example you can run through your terminal (assuming you've setup myria-python):
+We illustrate the basic functionality using examples in the directory `jsonQueries/getting_started`. The  `jsonQueries` directory contains additional examples. In the example below, we upload the smallTable to the Myria Service. Here is an example you can run in a terminal (assuming you've setup myria-python):
 
-```
+```shell
 myria_upload --overwrite --hostname demo.myria.cs.washington.edu --port 8753 --no-ssl --relation smallTable /path/to/file
 ```
 
 ### Part 2: Running MyriaQL Queries
```

Comment:
We need to agree on a consistent convention for referring to the Myria query language. Seems that "MyriaL" is used throughout recent code.